tests/playbooks/tests_route_next_hop_ethernet.yml

```diff
@@ -0,0 +1,188 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Test configuring ethernet devices
```

Comment:
The name is wrong, maybe a result from copying another test? Also, the name of the test file "next hop" - this term is not used in the code, warning or commit message. Seems more appropriate to call it `tests_route_device.yml`. Are there plans to test other devices than ethernet? If not, there is no need for the ethernet suffix.

Reply:
> The name is wrong, maybe a result from copying another test? 

Yes, honestly, I should be more smart about writing descriptive name. 

> Are there plans to test other devices than ethernet? If not, there is no need for the ethernet suffix.

Currently, there is no plan.